CONTRIBUTING.adoc

```diff
@@ -324,6 +324,13 @@ NOTE: If you check in out of date attributes the build will fail
 . Having create the PR, automated tests will run. If they fail, use the error log to determine the problem, and fix it
 . Once your build is passing ask for review from a Subject Matter Expert (who will check for accuracy), a writer (who will check that the content is up to expected quality for substance, formatting, style, structure and consistency), and a developer who will ensure the steps covered by the quick start are the end to end test suite.
 
+== Adding new product to repository
+
+. Go to ./docs/product-mapping.yaml file and add new product to the list of products.
```

Comment:
I'm not sure we want to put the product mapping in the root of the docs directory like that. I wonder if it should be a dotfile?

Reply:
`.product-mapping` sounds good (but yaml still preferred)